tutorial/lesson_15_generators_usage.sh

```diff
@@ -194,21 +194,21 @@ check_no_runtime()
     -f my_first_generator_basic \
```

Comment:
Probably need to add a note at the end of the comment above "Note that this tutorial assumes you're on linux x86-64. You may need to tweak the targets below if you're on a different platform."

Reply:
Does it? It doesn't look like any of the generated code actually runs.